Added MFA time sync solution to troubleshooting (#3722)
mattermost/docs: source/install/troubleshooting.rst

The password reset process is completed on the IdP provider side, and not via the Mattermost System Console.

Multi-Factor Authentication (MFA)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> If your MFA stopped working make sure your server clock not out of sync. On Unix systems you can show the hardware time of your system with ``# hwclock -r``.

Reviewer: ```suggestion
If MFA has stopped working, confirm that your server clock is not out of sync. On Unix systems you can show the hardware time of your system with ``# hwclock -r``.
```

Reviewer: Would MFA stop working on all devices, and would there be other steps if the server clock is in sync and MFA still isn't working?

Reviewer 2: I'd additionally check with timedatectl since it might show more information than hwclock and is (afaik) the preferred method on systemd enabled systems.

Author: AFAIK `timedatectl ` is for system time where `hwclock` is for the hardware clock that is used as seed for the Unix system time. 

Do you know if the users had issues with their hardware clock or did their system time run out of sync due to high server up-time and no time synchronization? I assumed the former, but in case of the latter we can advise to use `date` (which should be POSIX compatible?) WDYT @svelle ?

Reviewer 2: `hwclock` isn't really useful here, since mattermost doesn't check with the hardware clock at all.

![image](https://user-images.githubusercontent.com/7526550/86240510-a7595600-bba1-11ea-902e-34ed9dc08d43.png)

`date` would be more appropriate than `hwclock` but I'd still prefer timedatectl since it also shows RTC as well as info on time synchronization if enabled or not.

Author: @svelle Thanks for clarifying! This sounds like there is more then just checking local time. Do you know what information about the time is needed exactly in order to properly debug the issue?

> If your MFA stopped working make sure your server clock not out of sync. On Unix systems you can show the hardware time of your system with ``# hwclock -r``. If you are using systemd you can also use ``# timedatectl``.

Reviewer: ```suggestion
If MFA has stopped working, confirm that your server clock is not out of sync. On Unix systems you can view the hardware time of your system using ``# hwclock -r``. If you're using systemd you can also use ``# timedatectl``.
```
